### PR TITLE
Feature aimarketplace service listing responsive against footer-responsive

### PR DESCRIPTION
**Pull request by Vivek205**

### CODE REVIEW

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js [hunk 1]
```diff
@@ -21,9 +25,11 @@ export const useStyles = makeStyles(theme => ({
   cardSubheader: {
     color: theme.palette.text.darkShadedGray,
     fontWeight: theme.typography.fontweight,
-    fontSize: 20,
-    letterSpacing: 0.25,
+    fontSize: 18,
```
Comment (chetankothari) [inline src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js]: We could extract the font sizes into constants and have consistent typography. We could also consider building components for `Heading`, `Subheading`, `Description` and `Text` which have pre defined size and spacing. Will be much easier to maintain and will give a more consistent look and feel to the app.

File: src/components/UserProfile/UserProfileSettings/index.js [hunk 2]
```diff
@@ -71,7 +71,9 @@ class UserProfileSettings extends Component {
             </div>
             <div>
               <h4>Password</h4>
-              <Link>Change Password</Link>
+              <Link>
```
Comment (chetankothari) [inline src/components/UserProfile/UserProfileSettings/index.js]: Do you need Link here ?

File: src/components/common/StyledButton/index.js [hunk 3]
```diff
@@ -19,14 +20,14 @@ const StyledButton = ({ disabled, onClick, type, iconClass, btnText }) => {
 
   return (
     <Button className={clsx(classes.styledButton, classes[buttonColor[type]])} disabled={disabled} onClick={onClick}>
-      <Icon className={iconClass} />
+      {iconClass ? <Icon className={iconClass} /> : null}
```
Comment (chetankothari) [inline src/components/common/StyledButton/index.js]: Can this be handled within Icon class?

File: src/components/common/StyledTextField/styles.js [hunk 4]
```diff
@@ -1,10 +1,7 @@
...
   styledTextField: {
-    "& label": {
-      color: theme.palette.text.black1,
-    },
-    "& div": {
-      color: theme.palette.text.black1,
-    },
+    "& label": { color: theme.palette.text.black1 },
+    "& div": { color: "#212121 !important" },
+    "& fieldset": { borderColor: "#828282 !important" },
```
Comment (chetankothari) [inline src/components/common/StyledTextField/styles.js]: Do we need `!important`?

File: src/config/APIEndpoints.js [hunk 5]
```diff
@@ -0,0 +1,6 @@
+export const APIEndpoints = {
+    GET_SERVICE_LIST: {
+        name: "Get Service",
+        endpoint: "https://3zjz5tihg2.execute-api.us-east-1.amazonaws.com/ropsten",
```
Comment (chetankothari) [inline src/config/APIEndpoints.js]: Should this be coming from env and not be checked in?

File: src/config/aws_config.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    // OPTIONAL - if your API requires authentication
+    Auth: {
+        // REQUIRED - Amazon Cognito Identity Pool ID
+        identityPoolId: "us-east-1:8a98cb16-e64b-4aec-bcbb-30c447aec81a",
+        // REQUIRED - Amazon Cognito Region
+        region: "us-east-1",
+        // REQUIRED - Amazon Cognito User Pool ID
+        userPoolId: "us-east-1_65MHedYHj",
+        // REQUIRED - Amazon Cognito Web Client ID (26-char alphanumeric string)
+        userPoolWebClientId: "49d6u3538qjlet95rs2aoo9usj",
```
Comment (chetankothari) [inline src/config/aws_config.js]: Should these be coming from env and not be checked in?

File: src/utility/constants/APIEndpoints.js [hunk 7]
```diff
@@ -0,0 +1,6 @@
+export const APIEndpoints = {
+    GET_SERVICES_LIST: {
+        name: "Get Service",
+        endpoint: "https://3zjz5tihg2.execute-api.us-east-1.amazonaws.com/ropsten",
```
Comment (chetankothari) [inline src/utility/constants/APIEndpoints.js]: Should this be coming from env and not be checked in?

**Review by chetankothari — CHANGES_REQUESTED** [review]

File: src/components/common/StyledButton/index.js (hunk 3 above)
Comment (Vivek205) [reply src/components/common/StyledButton/index.js]: Nope we handled that way.. but it is taking a position in the DOM and causing problem in the styling of the button.

File: src/components/common/StyledTextField/styles.js (hunk 4 above)
Comment (Vivek205) [reply src/components/common/StyledTextField/styles.js]: Yh. I confirmed it with Ananya.

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js (hunk 1 above)
Comment (Vivek205) [reply src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js]: will speak with design team reg this...

File: src/config/APIEndpoints.js (hunk 5 above)
Comment (Vivek205) [reply src/config/APIEndpoints.js]: Deleted this file... already added this in gitIgnore. I don't know how it got commited

File: src/config/aws_config.js (hunk 6 above)
Comment (Vivek205) [reply src/config/aws_config.js]: Deleted this file... already added this in gitIgnore. I don't know how it got commited

File: src/utility/constants/APIEndpoints.js (hunk 7 above)
Comment (Vivek205) [reply src/utility/constants/APIEndpoints.js]: deleted

File: src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js (hunk 1 above)
Comment (Vivek205) [reply src/components/AiMarketplace/MainSection/ServiceCollection/CardGroup/StyledCard/styles.js]: connected to Greg reg this and created an [issue](https://github.com/singnet/refactored-dapp/issues/37)

File: src/components/UserProfile/UserProfileSettings/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/UserProfile/UserProfileSettings/index.js]: Yh chetan, we require that for routing to change password

**Review by chetankothari — APPROVED** [review]

File: src/components/common/StyledTextField/styles.js (hunk 4 above)
Comment (Vivek205) [reply src/components/common/StyledTextField/styles.js]: [raised issue](https://github.com/singnet/refactored-dapp/issues/39)

File: src/components/common/StyledButton/index.js (hunk 3 above)
Comment (Vivek205) [reply src/components/common/StyledButton/index.js]: [raised issue](https://github.com/singnet/refactored-dapp/issues/38)

File: src/components/UserProfile/UserProfileSettings/index.js (hunk 2 above)
Comment (Vivek205) [reply src/components/UserProfile/UserProfileSettings/index.js]: removed the link

**Pull request closed without merging**